queries.sql

```diff
@@ -154,3 +210,126 @@ FROM
     ON owners.id = animals.owner_id 
   GROUP BY owners.full_name 
   ORDER BY COUNT(animals.name) DESC LIMIT 1) number ;
+
+select * from visits where vets_id=1 order by date_of_visit desc limit 1;
+
+/*Who was the last animal seen by William Tatcher?*/
+SELECT
+  animals.name,
+  date_of_visit
+FROM
+  animals
+INNER JOIN visits
+  ON animals.id = animal_id
+WHERE
+  vets_id = (SELECT id FROM vets WHERE name LIKE 'William Tatcher')
+ORDER BY date_of_visit DESC LIMIT 1;
+
+/*How many different animals did Stephanie Mendez see?*/
+SELECT
+  count(animal_id) as number_of_animals
+FROM
+  visits
+WHERE
+  vets_id = (SELECT id FROM vets WHERE name LIKE 'Stephanie Mendez');
+
+-- List all vets and their specialties, including vets with no specialties.
+SELECT
+  vets.name as vet_name,
+  species.name as specilization
+FROM
+  vets
+LEFT JOIN specializations
+  ON vets.id = vets_id
+LEFT JOIN species
+  ON specializations.species_id = species.id;
+
+-- List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
+SELECT
+  animals.name as animal_name,
+  date_of_visit
+FROM
+  visits
+JOIN animals
+  ON animal_id = animals.id
+WHERE 
+  (date_of_visit BETWEEN '2020-04-01' AND '2020-08-30') AND
+  visits.vets_id = (SELECT id FROM vets WHERE vets.name = 'Stephanie Mendez')
+ORDER BY date_of_visit ASC;
+
+-- What animal has the most visits to vets?
+SELECT
+  animals.name as animal_name,
+  MAX(animal_id) as number_of_visits
+FROM
+  visits
+JOIN animals
+  ON animal_id = animals.id
+GROUP BY animals.name
+ORDER BY number_of_visits DESC LIMIT 1;
```

Comment:
Thanks for catching this, I am working on improving it, now that you mention it the data I got with my query can't be possible since we don't have that many visits data, I didn't notice that.

Reply:
I am done fixing it, thanks for the help again @Gambit142 .